### PR TITLE
Fix for Eval of EfficientNet.

### PR DESCRIPTION
**Pull request by kevinstephano**

### CODE REVIEW

**Comment by jjsjann123** [comment]
This is dangerous that we might accidentally ignore issues from profiling runs.
But in reality, we do check/assert for necessary constant while parsing. I think it's reasonable that we relax checks/asserts.

File: torch/csrc/jit/codegen/cuda/graph_fuser.cpp [hunk 1]
```diff
@@ -63,11 +63,12 @@ Value* createConditionalConstant(Node* profile_ivalue) {
         static_cast<bool>(profile_ivalue->i(Symbol::attr("profiled_bool"))));
   } else {
     GRAPH_DEBUG("profile_ivalue: ", *profile_ivalue);
-    TORCH_INTERNAL_ASSERT(
-        false,
-        __func__,
-        " gets unidentified type: ",
-        profile_ivalue->ty(attr::profiled_type));
+    return nullptr;
```
Comment (jjsjann123) [inline torch/csrc/jit/codegen/cuda/graph_fuser.cpp:71]: nitpick: maybe throw a warning here, this is still pretty surprising that we see un-profiled values.

**Review by jjsjann123 — COMMENTED** [review]
I think this one is in general a good thing to have.

Let me take a quick look at the original issue first.

**Comment by kevinstephano** [comment]
I didn't mean to imply this was the proper solution, this just fixed the issue at hand.

**Comment by jjsjann123** [comment]
@kevinstephano I updated it a little bit and tried it on the benchmark.

This reminds me about a refactor that I meant to do -> (I should separate partition & profiling API), but that's irrelevant to this quick fix.

**Comment by kevinstephano** [comment]
The change to remove the profile_ivalue node looks good. I figured you might know better than me.

**Comment by jjsjann123** [comment]
Thanks for checking. Just got local build + test passed. I'm merging this one

**Review by jjsjann123 — APPROVED** [review]
Stamping for merge

**Comment by jjsjann123** [comment]
I think I made a mistake in the commit. This PR should fix #874

**Pull request merged**